source/ClientRequest/AMDSClientRelativeCountPlusCountDataRequest.cpp

```diff
@@ -0,0 +1,76 @@
+#include "source/ClientRequest/AMDSClientRelativeCountPlusCountDataRequest.h"
+
+#include "source/AMDSDataStream.h"
+
+AMDSClientRelativeCountPlusCountDataRequest::AMDSClientRelativeCountPlusCountDataRequest(QObject *parent) :
+	AMDSClientDataRequest(parent)
+{
+	setRequestType(AMDSClientRequestDefinitions::RelativeCountPlusCount);
+
+	setRelativeCount(0);
+	setCount(1);
+}
+
+AMDSClientRelativeCountPlusCountDataRequest::AMDSClientRelativeCountPlusCountDataRequest(ResponseType responseType, const QString &socketKey, const QString &bufferName, bool includeStatusData, const quint64 relativeCount, quint64 count, const AMDSBufferGroupInfo &bufferGroupInfo, QObject *parent):
+	AMDSClientDataRequest(socketKey, QString(), AMDSClientRequestDefinitions::RelativeCountPlusCount, responseType, bufferName, includeStatusData, bufferGroupInfo, parent)
+{
+	setRelativeCount(relativeCount);
+	setCount(count);
+}
+
+AMDSClientRelativeCountPlusCountDataRequest::~AMDSClientRelativeCountPlusCountDataRequest()
+{
+
+}
+
+AMDSClientRelativeCountPlusCountDataRequest::AMDSClientRelativeCountPlusCountDataRequest(const AMDSClientRelativeCountPlusCountDataRequest &other):
+	AMDSClientDataRequest(0)
```

Comment:
It will be nice to pass `other` to the super class. But it doesn't have to for this case, since the `=` operator can guarantee that all the parents values are copied.